internal/distro/rhel85/package_sets.go

```diff
@@ -64,15 +64,22 @@ func x8664InstallerBuildPackageSet() rpmmd.PackageSet {
 
 // BOOT PACKAGE SETS
 
-// x86_64 arch-specific boot package set
-func x8664BootPackageSet() rpmmd.PackageSet {
+// x86_64 Legacy arch-specific boot package set
+func x8664LegacyBootPackageSet() rpmmd.PackageSet {
 	return rpmmd.PackageSet{
-		Include: []string{"dracut-config-generic", "grub2-pc", "grub2-efi-x64", "shim-x64"},
+		Include: []string{"dracut-config-generic", "grub2-pc"},
```

Comment:
isn't `dracut-config-generic` needed for all images? Otherwise, `dracut` builds `initramfs` specific for the host. Not sure where to put it though. Maybe in both legacy and uefi sets?

Reply:
Yeah, I was not sure. I can put it in both sets. Having a duplicated package should not affect the depsolved package set.

Reply:
sgtm, it will be less confusing if we ever want to build uefi-only images.

Reply:
Let's fix in a follow-up.